src/helpers/env.ts

```diff
@@ -1,7 +1,43 @@
-export function isStaging() {
-  return process.env.NEXT_PUBLIC_ENV === 'staging';
-}
+'use client';
+
+export const IS_FREE_PLAN: boolean = process.env.NEXT_PUBLIC_ENV === 'staging' && window.origin === 'https://free.maffin.io';
+export const IS_PAID_PLAN: boolean = process.env.NEXT_PUBLIC_ENV === 'master';
+export const IS_DEMO_PLAN: boolean = !IS_FREE_PLAN && !IS_PAID_PLAN;
+
+const PAID_PLAN_CONFIG = {
+  auth0: {
+    domain: 'maffin.eu.auth0.com',
+    clientId: 'cEXnN96kEP3ER2EDJjmjRW0u2MEFBUKK',
+    scopes: 'profile email https://www.googleapis.com/auth/drive.file',
+  },
+};
+
+const FREE_PLAN_CONFIG = {
+  auth0: {
+    domain: 'maffin-dev.eu.auth0.com',
+    clientId: 'mMmnR4NbQOnim9B8QZfe9wfFuaKb8rwW',
```

Comment:
yes, we use dev for pricing plans. Dunno if it's temporary or not but it is to avoid breaking prod when doing changes